spec/models/user_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  it 'is valid with a name' do
+    user = User.new(name: 'John Doe')
+    expect(user).to be_valid
+  end
+
+  it 'is invalid without a name' do
+    user = User.new(name: '')
+    expect(user).to_not be_valid
+    expect(user.errors[:name]).to include('can not be blank')
+  end
```

Comment:
- [ ] Please also add tests for  your Posts Counter `PostsCounter must be an integer greater than or equal to zero`, Kindly ensure that all unit  tests written are passing for all your models ✅